content/tutorials/mqtt-snake.textile

```diff
@@ -38,6 +38,8 @@ blang[go].
 blang[python].
   h4. The complete source code for each step of "this tutorial is available on Github":https://github.com/ably/tutorials/commits/mqtt-snake-python.
 
+  *Note*: The Python version of this tutorial requires "Curses":https://docs.python.org/3/howto/curses.html to be installed. The library will only be functional on *Unix based machines*, and will not work at all *Windows*.
```

Comment:
Possible improvement (to sound a little smoother):

*Note*: The Python version of this tutorial requires the installation of "Curses":https://docs.python.org/3/howto/curses.html library. However, this library will only work on Unix-based machines, not *Windows*.

Reply:
Thanks @franrob-projects  I've updated it.